src/modules/autocomplete/autocomplete.js

```diff
@@ -96,7 +96,8 @@ function createAutocomplete(window) {
                     templates: {
                         inputValue: inputValueTemplate,
                         suggestion: suggestionTemplate
-                    }
+                    },
+                    tAssistiveHintVisible: true
```

Comment:
What does `tAssistiveHintVisible` do?

`visible` implies that it can be hidden but still there. Is this the intended meaning behind this variable name?

Reply:
Set this to false and it will be hidden so yes. Set this variable and it becomes visible, styled and above the input.